index.html

```diff
@@ -7,9 +7,27 @@
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+ <header class="book-header"> 
+ <h1 class="book-title">Título del libro</h1>
+ <p class="book-genre">(género)</p>
+ </header>
+ <div class = "book-content">
+   <div class="book-content-row">
+      <h3 class="book-author-title">Autor del libro: </h3>
```

Comment:
.book-author-title, book-editorial-title and book-publish-date-title can be the same class because this are similar, here you are creating unnecessary classes same for book-editorial-name, book-publish-date and book-author-name